Formula/pipx.rb

```diff
@@ -61,6 +61,23 @@ def install
     (bin/"pipx").write_env_script(libexec/"bin/pipx", PYTHONPATH: ENV["PYTHONPATH"])
     (bin/"register-python-argcomplete").write_env_script(libexec/"bin/register-python-argcomplete",
       PYTHONPATH: ENV["PYTHONPATH"])
+
+    # Install shell completions
+    output = Utils.safe_popen_read(libexec/"bin/register-python-argcomplete", "--shell=bash", "pipx")
+    (bash_completion/"pipx").write output
+
+    output = Utils.safe_popen_read(libexec/"bin/register-python-argcomplete", "--shell=fish", "pipx")
+    (fish_completion/"pipx.fish").write output
+
+    output = Utils.safe_popen_read(libexec/"bin/register-python-argcomplete", "--shell=bash", "pipx")
+    (zsh_completion/"_pipx").write "#compdef pipx\n#{output}"
+  end
+
+  def caveats
+    <<~EOS
+      To enable zsh completions, add the following to your ~/.zshrc:
+        autoload -U bashcompinit && bashcompinit
+    EOS
   end
```

Comment:
I think we're better off not shipping ZSH then

Reply:
makes sense, updated